src/nick-bot-commands.js

```diff
@@ -0,0 +1,47 @@
+import {findCommandFiles, getRestInstance} from "./shared.js";
```

Comment:
rename the file to `bot-commands.js` to make the code / files the same on both bots (and possible future bots)?